lib/config.ts

```diff
@@ -395,6 +399,10 @@ const calculateValue = () => {
         redis: {
             url: envs.REDIS_URL || 'redis://localhost:6379/',
         },
+        otel: {
+            seconds_bucket: envs.OTEL_SECONDS_BUCKET || '0.01,0.1,1,2,5,15,30,60',
+            milliseconds_bucket: envs.OTEL_MILLISECONDS_BUCKET || '10,20,50,100,250,500,1000,5000,15000',
+        },
```

Comment:
Please place it under the section https://github.com/DIYgod/RSSHub/blob/e5a4069a22ac5e705ca57e2c6892700a910aab32/lib/config.ts#L420-L429

Reply:
changed